src/components/LeftSidebar/Sponsors.astro

```diff
@@ -5,22 +5,14 @@ import UIString from '../UIString.astro';
-			></path>
+		<svg xmlns="http://www.w3.org/2000/svg" width="90px" viewBox="0 0 256 105"
+			><g clip-path="url(#a)"
+				><path
+					d="M58.47 103.765v-26.35l.547-.547h6.587l.546.546v26.351l-.546.546h-6.587l-.547-.546ZM58.47 26.897V.547L59.017 0h6.587l.546.546v26.351l-.546.546h-6.587l-.547-.546ZM35.797 85.24h-.904l-4.531-4.532v-.904l8.49-8.5 4.796.01.64.63v4.796l-8.49 8.5ZM30.362 24.747v-.913l4.53-4.523h.905l8.491 8.49v4.788l-.64.648h-4.796l-8.49-8.49ZM.546 48.307H37.88l.546.546v6.597l-.547.546H.547L0 55.45v-6.597l.546-.546ZM255.445 48.316l.546.546v6.588l-.546.546h-37.879l-.546-.546 2.739-6.588.546-.546h35.14Z"
+					fill="#32E6E2"></path><path
+					d="M74.667 65.886h-6.588l-.546-.546V49.92c0-2.748-1.075-4.873-4.386-4.94-1.707-.043-3.653 0-5.735.085l-.316.315v19.951l-.546.546h-6.587l-.547-.546v-26.35l.547-.547h14.822c5.76 0 10.428 4.668 10.428 10.428v16.47l-.546.545v.009ZM106.573 54.349l-.546.546H88.994l-.546.546c0 1.1 1.1 4.395 5.496 4.395 1.646 0 3.293-.546 3.848-1.647l.546-.546h6.588l.546.546c-.546 3.294-3.294 8.243-11.537 8.243-9.335 0-13.73-6.588-13.73-14.285S84.6 37.862 93.389 37.862s13.184 6.588 13.184 14.285v2.202Zm-8.243-5.496c0-.546-.546-4.394-4.941-4.394s-4.941 3.848-4.941 4.394l.546.547h8.79l.546-.547ZM121.95 57.643c0 1.1.546 1.647 1.647 1.647h4.941l.546.546v5.495l-.546.546h-4.941c-4.941 0-9.336-2.201-9.336-8.243V45.551l-.546-.546h-3.848l-.546-.546v-5.496l.546-.546h3.848l.546-.546v-4.94l.546-.547h6.588l.546.546v4.94l.546.547h6.042l.546.546v5.496l-.546.546h-6.042l-.546.546v12.083l.009.009ZM142.276 65.886h-6.587l-.547-.546V27.98l.547-.545h6.587l.546.546v37.35l-.546.546v.009ZM157.107 34.022h-6.588l-.546-.546v-5.495l.546-.546h6.588l.546.546v5.495l-.546.546Zm0 31.864h-6.588l-.546-.546V38.972l.546-.546h6.588l.546.546V65.34l-.546.546ZM182.929 27.98v5.496l-.546.546h-4.941c-1.101 0-1.647.547-1.647 1.647v2.202l.546.546h5.496l.546.546v5.496l-.546.546h-5.496l-.546.546v19.772l-.546.546h-6.588l-.546-.546V45.55l-.546-.546h-3.848l-.547-.546v-5.496l.547-.546h3.848l.546-.546v-2.202c0-6.041 4.395-8.243 9.336-8.243h4.94l.547.546-.009.009ZM203.247 66.432c-2.202 5.495-4.395 8.79-12.083 8.79h-2.748l-.546-.547V69.18l.546-.546h2.748c2.747 0 3.294-.546 3.848-2.194v-.546l-8.789-21.427v-5.495l.546-.546h4.941l.546.546 6.588 18.68h.546l6.588-18.68.546-.546h4.941l.546.546v5.495l-8.79 21.974.026-.009Z"
+				></path>
+			</g><defs><clipPath id="a"><path d="M0 0h256v104.311H0z"></path></clipPath></defs>
 		</svg>
```

Comment:
Same here, plus important to add `aria-hidden="true"` here to hide the SVG — we provide the accessible text via the `aria-label` attribute on the link. I also felt the new logo felt smaller at 90px wide compared to the Storyblok one, so bumped that up to 105px.

```suggestion
		<svg xmlns="http://www.w3.org/2000/svg" width="105" viewBox="0 0 256 105" aria-hidden="true">
			<path fill="#32E6E2" d="M58.5 103.8V77.4l.5-.5h6.6l.6.5v26.4l-.6.5H59l-.5-.5Zm0-76.9V.5L59 0h6.6l.6.5V27l-.6.5H59l-.5-.5ZM35.8 85.2h-1l-4.4-4.5v-.9l8.5-8.5h4.7l.7.6v4.8l-8.5 8.5Zm-5.4-60.5v-.9l4.5-4.5h.9l8.5 8.5v4.8l-.7.6H39l-8.5-8.5ZM.5 48.3H38l.5.6v6.6l-.5.5H.5l-.5-.5v-6.6l.5-.6Zm255 0 .5.6v6.6l-.6.5h-37.8l-.6-.5 2.8-6.6.5-.6h35.1Z"></path>
			<path d="M74.7 65.9H68l-.6-.6V50c0-2.7-1-4.9-4.4-5-1.7 0-3.6 0-5.7.2l-.3.3v20l-.6.5H50l-.6-.6V39l.6-.6h14.8c5.7 0 10.4 4.7 10.4 10.5v16.4l-.5.6Zm31.9-11.6-.6.6H89l-.6.5c0 1.1 1.1 4.4 5.5 4.4 1.7 0 3.3-.5 3.9-1.6l.5-.6h6.6l.6.6c-.6 3.3-3.3 8.2-11.6 8.2-9.3 0-13.7-6.6-13.7-14.3S84.6 38 93.4 38s13.2 6.6 13.2 14.2v2.2ZM98.3 49c0-.6-.5-4.4-5-4.4s-4.9 3.8-4.9 4.4l.6.5h8.8l.5-.5Zm23.7 8.7c0 1.1.5 1.7 1.6 1.7h5l.5.5v5.5l-.6.6h-4.9c-5 0-9.3-2.2-9.3-8.3v-12l-.6-.6H110l-.6-.5V39l.6-.6h3.8l.6-.5v-5l.5-.5h6.6l.5.5v5l.6.5h6l.6.6v5.5l-.6.5h-6l-.6.6v12Zm20.3 8.3h-6.6l-.6-.6V28l.6-.6h6.6l.5.6v37.3l-.5.6ZM157 34h-6.6l-.5-.5V28l.5-.6h6.6l.6.6v5.5l-.6.5Zm0 31.9h-6.6l-.5-.6V39l.5-.6h6.6l.6.6v26.3l-.6.6Zm25.8-38v5.6l-.5.5h-5c-1 0-1.6.6-1.6 1.7v2.2l.5.5h5.5l.6.6v5.5l-.6.5h-5.5l-.5.6v19.7l-.6.6h-6.5l-.6-.6V45.5l-.5-.5h-3.9l-.5-.5V39l.5-.6h3.9l.5-.5v-2.2c0-6 4.4-8.3 9.4-8.3h4.9l.5.6Zm20.3 38.5c-2.2 5.5-4.3 8.8-12 8.8h-2.8l-.5-.5v-5.5l.5-.6h2.8c2.7 0 3.3-.5 3.8-2.2V66l-8.8-21.4V39l.6-.6h5l.5.6 6.5 18.7h.6L206 39l.5-.6h5l.5.6v5.5l-8.8 22Z"></path>
		</svg>
```

Reply:
Cool, I thought it was small compared to the old one and next to Storyblok but wasn't sure if you all would want it bumped. Thanks!